plugins/spy.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: spy
```

Comment:
In my opinion it's unique enough that people probably won't try to create another spy plugin. Thats said, the name tells what the plugin does only a little bit, so this will impact its discoverability big time.

Reply:
Thanks @ahmetb and @corneliusweig so much for your reviews and improvement suggestions. I have updated the description with more accurate wording accordingly.

For the name selection of spy, I prefer this name because it is short and gives the impression that all actions will be non-invasive - that is the characteristic I try to differentiate with other debugging plugins. And I think it is also understandable for a debugging purpose.

Reply:
Alright, then let's stick with `spy`.